_includes/try-it.md

```diff
@@ -249,99 +324,58 @@ provisioned and associated with a `Machine`.
+The default username for the CentOS image is `centos`.
 
 ```sh
-ssh centos@192.168.111.21
+$ ssh centos@192.168.111.21
 ```
 
-Deprovisioning is done just by deleting the `Machine` object.
+##  Deprovisioning Cluster and Machines
 
-```sh
-$ kubectl delete machine centos -n metal3
-
-machine.cluster.k8s.io "centos" deleted
-```
-
-At this point you can see that the `BareMetalHost` is going through a
-deprovisioning process.
+Deprovisioning of the cluster and machines is done just by deleting `Cluster`
+`Machine` objects.
 
 ```sh
-$ kubectl get baremetalhosts -n metal3
-
-NAME       STATUS   PROVISIONING STATUS   MACHINE   BMC                         HARDWARE PROFILE   ONLINE   ERROR
-node-0     OK       ready                           ipmi://192.168.111.1:6230   unknown            true
-node-1     OK       deprovisioning                  ipmi://192.168.111.1:6231   unknown            false
-```
-
-## Provisioning a Cluster
-
-The v1alpha2 deployment can be done with Ubuntu 18.04 or Centos 7 target host
-images.
-
-### Requirements
-
-#### Dev env size
-
-The requirements for the dev env machine are, when deploying **Ubuntu** target
-hosts:
-
-* 16GB of memory
-* 4 cpus
-
-And when deploying **Centos** target hosts:
+$ kubectl delete machine test1-md-0-m87bq -n metal3
+machine.cluster.x-k8s.io "test1-md-0-m87bq" deleted
 
-* 32GB of memory
-* 4 cpus
+$ kubectl delete machine test1-controlplane-0 -n metal3
+machine.cluster.x-k8s.io "test1-controlplane-0" deleted
 
-The Minikube machine is deployed with 4GB of RAM, and 2 vCPUs, and the target
-hosts with 4 vCPUs and either 4GB of RAM (Ubuntu) or 8GB of RAM (Centos).
-
-### Environment variables
-
-The following environment variables need to be set for **Centos**:
-
-```sh
-export IMAGE_CHECKSUM=http://172.22.0.1/images/centos-updated.qcow2.md5sum
-export IMAGE_NAME=centos-updated.qcow2
-export CAPI_VERSION=v1alpha2
-export IMAGE_URL=http://172.22.0.1/images/centos-updated.qcow2
-export IMAGE_OS=Centos
-export DEFAULT_HOSTS_MEMORY=8192
+$ kubectl delete cluster test1 -n metal3
+cluster.cluster.x-k8s.io "test1" deleted
 ```
 
-And the following environment variables need to be set for **Ubuntu**:
+At this point you can see that the `BareMetalHost` and `Cluster` are going
+through a deprovisioning process.
 
 ```sh
-export CAPI_VERSION=v1alpha2
-export IMAGE_OS=Ubuntu
-export DEFAULT_HOSTS_MEMORY=4096
-```
-
-### Deploy the metal3 Dev env
+$ kubectl get baremetalhosts -n metal3
+NAME     STATUS   PROVISIONING STATUS   CONSUMER               BMC                         HARDWARE PROFILE   ONLINE   ERROR
+node-0   OK       deprovisioning        test1-md-0-m87bq       ipmi://192.168.111.1:6230   unknown            false     
+node-1   OK       deprovisioning        test1-controlplane-0   ipmi://192.168.111.1:6231   unknown            false
 
-```sh
-./01_prepare_host.sh
-./02_configure_host.sh
-./03_launch_mgmt_cluster.sh
+$ kubectl get cluster -n metal3
```

Comment:
Add a text above this to introduce the cluster (probably why it is also deprovisioning)